hw/bsp/dialog_da1469x-dk-pro/board.mk

```diff
@@ -0,0 +1,61 @@
+ CFLAGS += \
+  -flto \
+  -mthumb \
+  -mthumb-interwork \
+  -mabi=aapcs \
+  -mcpu=cortex-m33+nodsp \
+  -mfloat-abi=hard \
+  -mfpu=fpv5-sp-d16 \
+  -nostdlib \
+  -DCORE_M33 \
+  -DCFG_TUSB_MCU=OPT_MCU_DA1469X \
+  -DCFG_TUD_ENDPOINT0_SIZE=8\
+
+# mcu driver cause following warnings
+# CFLAGS += -Wno-error=strict-prototypes -Wno-error=unused-parameter
+
+MCU_FAMILY_DIR = hw/mcu/dialog/da1469x
+MCU_DIR = hw/mcu/dialog/da14699
+
+# All source paths should be relative to the top level.
+LD_FILE = hw/bsp/$(BOARD)/da1469x.ld
+
+SRC_C += \
+	$(MCU_FAMILY_DIR)/src/system_da1469x.c \
+	$(MCU_FAMILY_DIR)/src/da1469x_clock.c \
+	$(MCU_FAMILY_DIR)/src/hal_gpio.c \
+
+SRC_S += $(TOP)/hw/bsp/$(BOARD)/gcc_startup_da1469x.S
+
+INC += \
+    $(TOP)/hw/bsp/$(BOARD) \
+	$(TOP)/$(MCU_DIR)/include \
+	$(TOP)/$(MCU_FAMILY_DIR)/include \
+	$(TOP)/$(MCU_FAMILY_DIR)/SDK_10.0.8.105/sdk/bsp/include \
+
+# For TinyUSB port source
+VENDOR = dialog
+# While this is for da1469x chip, there is chance that da1468x chip family will also work
+CHIP_FAMILY = da146xx
+
```

Comment:
please also add following line for freeRTOS ports, I have tested, and it seems that cdc_msc_freertos works well enough

```
# For freeRTOS port source
FREERTOS_PORT = ARM_CM33_NTZ/non_secure
```